cms/templates/js/move-xblock-list.underscore

```diff
@@ -35,6 +31,17 @@
                     <span class="icon fa fa-arrow-right forward-sr-icon" aria-hidden="true"></span>
                     <span class="sr forward-sr-text"><%- gettext("Click for children") %></span>
                 </button>
+            <% } else { %>
```

Comment:
"View child components" (5 syllables) is still the best option IMO, perhaps "View child items" to lose one syllable. @cptvitamin?